temoa-uiuc/temoa-uiuc-chws.md

```diff
@@ -0,0 +1,49 @@
+# TEMOA's model of the CHWS
+
+* ExistingCapacity [Done]
+* Efficiency [Done]
+* CostInvest: No Invest Cost because UofI already has it
+* CostFixed: To be added in a future PR
+* CapacityToActivity [Done]
+* CapacityFactorTech [To ask]
+* Storage duration [To ask]
+
+# ExistingCapacity
+
+* 1 ton of chilled water ~ 3.5 kWh [1]
+* COP ~ 5 [2]
+* tons to kg ~ 1000
+* [tons]/(1000 * 5) * 3.5 = [MWe]
+* for example: 52,000 tons yield 36 MW
+* UIUC: 37,500 tons
+* UIUC's total capacity: 6.5e6 millon-gallon ~ 24,600 tons
+* capacity: 37,500 tons
+* time to discharge: 39.4 min
+* electrical-power: 37,500 tons/5e3 * 3.5 = 26.2 MW
+* total electrical-capacity: 26.2 MW * 39.4 min/60 min * 1h = 17.2 MWh
+
+# Efficiency:
+
+* Ideally, we should separate the efficiency of converting electricity into CHW and CHW to electricity.
+* The former is probably slightly lower than the latter.
+* As both are close to 1 anyway, we will consider an efficiency of 1.
```

Comment:
Does it really take 1 unit of electricity to produce 1 unit of chilled water?
That doesn't seem correct...